### PR TITLE
Changed the badges and case.

### LINKED ISSUE #1: Readme badges
Adding some informative badges into README.md

such as: 
[](https://pypi.org/project/squiral/) [](https://pypi.org/project/squiral/)

### LINKED ISSUE #2: Basic documentation
Adding basic documentation (such as install, usage, etc) into README.md

### PR DESCRIPTION
**Pull request by nikitamahoviya**
Fixes #1 and Fixes #2

### CODE REVIEW

**Comment by sadikkuzu** [comment]
Thanks again for your contribution! 
Without people like you submitting PRs, open source projects couldn't be run.
You rock, @nikitamahoviya! 🚀

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # squiral
+![](https://img.shields.io/pypi/v/squiral) [![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/) ![]( https://img.shields.io/pypi/pyversions/squiral.svg )
```
Comment (sadikkuzu) [inline README.md:2]: Very well done!

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "squiral"
-version = "0.1.1"
```
Comment (sadikkuzu) [inline pyproject.toml:3]: Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/pyproject.toml#L3

File: squiral/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-__version__ = '0.1.1'
```
Comment (sadikkuzu) [inline squiral/__init__.py:1]: Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/squiral/__init__.py#L1

File: tests/test_squiral.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 
 def test_version():
-    assert __version__ == '0.1.1'
```
Comment (sadikkuzu) [inline tests/test_squiral.py:8]: Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/tests/test_squiral.py#L8

**Review by sadikkuzu — CHANGES_REQUESTED** [review]
I've added my review as inline comments.
I guess your PR is based on one of previous commits.
Thanks again.

**Comment by sadikkuzu** [comment]
Thanks again @nikitamahoviya ✨

**Comment by nikitamahoviya** [comment]
> Thanks again @nikitamahoviya ✨

Hey!!
Thank you so much😃
I am sorry I couldn't look into it once again as I just got engaged in my GSoC project.
It was a great experience working with you!!

**Pull request merged**